src/controls/registerDefaultControls.js

```diff
@@ -80,7 +82,17 @@ function registerDefaultControls(controls, element, opts) {
   };
 
 
-  var enabledControls = [ 'scrollZoom', 'touchView', 'pinch' ];
+  switch (opts.dragMode) {
+    case 'pinch':
+       controlMethods.pinch = new DragControlMethod(element, 'touch', { hammerEvent: 'pinch'});
+      break;
+    default:
```

Comment:
Write an error check in the `default` case, like we do for `opt.mouseViewMode` below.